Add Render deployment guide

jess-render opened the pull request:
## About

This PR adds a Render deployment guide for the Prisma ORM. 
It pairs with the example code in https://github.com/prisma/prisma-examples/pull/6990

jharrell commented on content/200-orm/200-prisma-client/500-deployment/101-traditional/225-deploy-to-render.mdx:
```diff
@@ -0,0 +1,99 @@
+metaDescription: 'Learn how to deploy a Node.js server that uses Prisma ORM to Render.'
+---
+
+<TopBlock>
```
This is a deprecated component. not needed anymore 😊 
```suggestion
```

jharrell commented on content/200-orm/200-prisma-client/500-deployment/101-traditional/225-deploy-to-render.mdx:
```diff
@@ -0,0 +1,99 @@
+- Render natively supports [common runtimes](https://docs.render.com/language-support), including Node.js and Bun. In this guide, we'll use the Node.js runtime.
+- Render [integrates with Git repos](https://docs.render.com/github) for automatic deployments upon commits. You can deploy to Render from GitHub, GitLab, or Bitbucket. In this guide, we'll deploy from a Git repository.
+
+</TopBlock>
```
```suggestion
```

jharrell commented on content/200-orm/200-prisma-client/500-deployment/101-traditional/225-deploy-to-render.mdx:
```diff
@@ -0,0 +1,99 @@
+- Sign up for a [Render](https://render.com) account
+
+## Get the example code
+Download the [example code](https://github.com/prisma/prisma-examples/tree/latest/deployment-platforms/render) to your local machine.
```
````suggestion
Download the [example code](https://github.com/prisma/prisma-examples/tree/latest/deployment-platforms/render) to your local machine.
```terminal
curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=2 prisma-examples-latest/deployment-platforms/render
cd render
```
````

jharrell commented on content/200-orm/200-prisma-client/500-deployment/101-traditional/225-deploy-to-render.mdx:
```diff
@@ -0,0 +1,99 @@
+### Prisma schema and migrations
+
+The Prisma components of this app are in two files:
+- `prisma/schema.prisma`: The data model of this app. This example defines two models, `User` and `Post`. The format of this file follows the [Prisma schema](https://www.prisma.io/docs/orm/prisma-schema/overview).
+- `prisma/migrations/<migration name>/migration.sql`: The SQL commands that construct this schema in a PostgreSQL database. You can auto-generate migration files like this one by running [`prisma migrate dev`](https://www.prisma.io/docs/orm/prisma-migrate/understanding-prisma-migrate/mental-model#what-is-prisma-migrate).
```
```suggestion
- `prisma/schema.prisma`: The data model of this app. This example defines two models, `User` and `Post`. The format of this file follows the [Prisma schema](/orm/prisma-schema/overview).
- `prisma/migrations/<migration name>/migration.sql`: The SQL commands that construct this schema in a PostgreSQL database. You can auto-generate migration files like this one by running [`prisma migrate dev`](/orm/prisma-migrate/understanding-prisma-migrate/mental-model#what-is-prisma-migrate).
```

jharrell reviewed (commented):
Few small fixes needed. 

I'll make sure this is merged after the example is (needed for our dead link checker to work.)

jess-render commented:
Diffs since your review:
* Added a section about seeding the database, based on changes here https://github.com/prisma/prisma-examples/pull/6990#discussion_r1831104273
* Added Render to the list of traditional servers platforms in "Deployment paradigms", and linked to that page from the new guide.
